browser/help/topics/v4-hts.md

```diff
@@ -128,7 +128,7 @@ Global fields:
   - `vep_config`: VEP configuration to run VEP version with [Hail](https://hail.is/docs/0.2/methods/genetics.html#hail.methods.vep). File created using command within VEP init shell script in https://github.com/broadinstitute/gnomad_methods/tree/main.
   - `gencode_version`: GENCODE version used in VEP.
   - `mane_select_version`: MANE select version used in VEP.
-- `frequency_README`: Explanation of how to use the 'freq_index_dict' global annotation to extract frequencies from the 'freq' row annotation.
+- `frequency_README`: Explanation of how to use the '`freq_index_dict`' global annotation to extract frequencies from the '`freq`' row annotation.
```

Comment:
What's the difference between '`freq`' and `freq`? I saw from L335, you only changed to `genomes.freq`.

Reply:
Julia added these, but my thinking was that we add quotes here ('`freq`' ) because we're referring both to the name of the annotation and the actual annotation. to refer to `genomes.freq` in the same way, I think we'd want to do
> '`freq'` annotation within the '`genomes`' struct